rabbit-escape-render/src/rabbitescape/render/Vector2D.java

```diff
@@ -0,0 +1,99 @@
+package rabbitescape.render;
+
+import rabbitescape.engine.CellularDirection;
+
+public class Vector2D
+{
```

Comment:
Rabbit Escape contains 3 classes called `Point` already, 2 of which are identical :-)

However, I think `Vector2D` is a legitimate addition here since the methods on it don't make sense for those other classes.  As a side note, at least 2 of the `Point` classes should be combined.